### ISSUE TITLE
Multiple cards with the same name: new card is "silently discarded"

### ISSUE THREAD

**Issue by IzzySoft** [TheLastProject/Catima#103]
For one location I just wanted to add a second card. I edited the one already stored and wrote "old" into the comment. Then I scanned the new card, gave it the same name (that of the location), and put "new" into the comment. Hit save.

Well, nothing was stored – the new card was silently discarded. Mostlikely because another card by the same name already existed, and the name is somehow used as "primary key" / "id".

Instead of silently discarding, there should be an interaction, like "a card by this name already exists. Overwrite/Cancel/Discard(/Merge)" – with "Cancel" giving the chance to edit the name (and "Merge" adding it as a second code for the existing entry, should that become available).

**Comment by TheLastProject** [comment]
Nope, the name of the card is not the primary ID at all and multiple cards with the same name should be possible. But let's see if I can reproduce this.

**Comment by TheLastProject** [comment]
Hmm, nope, I can't reproduce it. Did you make sure to hit save in the bottom right? This may just be issue #17? If not, I'm afraid I'm going to need more detailed step-by-step instructions.

**Comment by IzzySoft** [comment]
I didn't try to reproduce, that's right, so it might have been a fluke: the second time I just gave it a different name, but AFAIR performed the same steps (and it worked). I tried to reproduce now, but can't either – so it looks like I was drawing conclusions too fast…

I don't think I hit the "back" arrow – but now that I cannot reproduce, I can't deny it might be possible. Well, let's close this as a duplicate to #17 for know. Should I find a way to reproduce, we can always re-open, right? And if I cannot reproduce: apologies for the noise!
